### PR TITLE
ML Changes and Fixes

### PR DESCRIPTION
**Pull request by gerryzqx**
- Before training the model, both the image and test image must be resized to the same size before passing it through to the CNN.
- Fixed a bug where when running the RealTime Prediction, the statistic window for the specific class, **point** would have a confidence percentage over 100%. The calculation for the confidence percentage now no longer go pass 100%.

### Before
Before

### After
After

### CODE REVIEW

File: ml/README.md [hunk 1]
```diff
@@ -6,21 +6,30 @@ This is a simple application of Convolution Neural Networks combined with backgr
...
+- Pillow (PIL) (6.2.0)
+- Imutils (0.5.3)
+
+## Installation
+
+`requirements.txt` specifies the required packages needed to run this project.
+To install these packages, you must have **pip** installed on your machine.
+
+```bash
+pip install -r requirements.txt
```
Comment (cpanican) [inline ml/README.md]: `pip` should be `pip3` since we will be using python3.

File: ml/README.md [hunk 2]
```diff
@@ -44,6 +53,10 @@ The model achieves an accuracy of **96.6%** on the validation dataset.
 
 The ratio of training set to validation set is **1000 : 100**.
 
+## How to train the model
+
+To train your model, open `ModelTrainer.ipynb` in jupyter notebook to train your model. If you don't have jupyter notebook installed, you can convert .ipynb to .py file by running this command in your terminal `ipython nbconvert --to python ModelTrainer.ipynb`. Now you can just run `python3 ModelTrainer.py`.
```
Comment (cpanican) [inline ml/README.md]: How does `nbconvert` work? Does it generate a new python file?

**Review by cpanican — CHANGES_REQUESTED** [review]
Overall it looks good just some small change on the readme file.

File: ml/README.md (hunk 1 above)
Comment (gerryzqx) [reply ml/README.md]: Yes, I had pip3 alias set to pip on my machine. I forgot I did that. I'll edit the read me for better simplicity.

File: ml/README.md (hunk 2 above)
Comment (gerryzqx) [reply ml/README.md]: Yes, nbconvert converts the ipynb file into a python file.

**Review by cpanican — APPROVED** [review]
